diff --git a/ui/gl/gl_detection.cpp b/ui/gl/gl_detection.cpp
index 24d330d..1a9becd 100644
--- a/ui/gl/gl_detection.cpp
+++ b/ui/gl/gl_detection.cpp
@@ -71,8 +71,8 @@ base::options::toggle OptionUseQtRhi({
 	.description = "Render the main window on the GPU instead of the CPU.",
 	.defaultValue = true,
 	.scope = [] {
-		return (!Platform::IsWindows() || Platform::IsWindowsARM64())
-			&& QLibraryInfo::version() >= QVersionNumber(6, 7);
+		// Windows x64 was excluded only because those builds were Qt 5.
+		return QLibraryInfo::version() >= QVersionNumber(6, 7);
 	},
 	.restartRequired = true,
 });
